projects/js-packages/components/components/icons/style.module.scss

```diff
@@ -0,0 +1,34 @@
+@import '@automattic/jetpack-base-styles/gutenberg-base-styles';
```

Comment:
I'm not sure whether this is still relevant with the way we're importing styles. Everywhere we're using this, `gutenberg-base-styles` will be included elsewhere.

We could maybe add this to the main package stylesheet instead?

Reply:
I'm not sure about this. @renatoagds maybe can help us here

Reply:
@pablinos Yes, it is better to remove it from here. This kind of style import should happen only at consumers (`plugin` or `package`).